src/get-start-and-end-commands.js

```diff
@@ -175,7 +172,7 @@ function runEmberRemotely({
   let isGlimmer = blueprint.packageName === glimmerPackageName && blueprint.name === glimmerPackageName;
 
   if (isCustomBlueprint || isGlimmer) {
-    args = [`ember-cli@${lastNode8Version}`, ...args];
+    args = ['ember-cli', ...args];
```

Comment:
I might do 'ember-cli@latest' here

Reply:
Yea I agree! that's what I did in my other PR that actually fixes the v2 addon blueprint: https://github.com/ember-cli/ember-cli-update/pull/1240/files#diff-b036a2bfff3e6c44af88056b383d9acaeb4f45c37efb5559d60ebec68250eb10L178